tools/apiview/emitters/cadl-apiview/.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
```

Comment:
Wes should try to avoid overriding these files from the root.

Reply:
Got it. I will try and merge these responsibly :)  Let me know if I screw it up!

Reply:
Updated, but let me know if you think there will be issues with how I updated.